templates/webapi/comments/comment_row.html.ep

```diff
@@ -15,27 +15,33 @@
             <div class="well well-lg">
                 <form onsubmit="updateComment(this); return false;" onreset="hideCommentEditor(this); return true;" data-put-url="<%= url_for($put_action => ($context->{type}.'_id' => $context->{id}, comment_id => $comment_id)) %>">
                     % if (current_user && current_user->is_admin) {
-                        <button class="btn btn-danger btn-circle btn-sm remove-edit-button" type="button" name="removeComment" onclick="deleteComment(this);" data-author="<%= $user->name %>" data-delete-url="<%= url_for($delete_action => ($context->{type}.'_id' => $context->{id}, comment_id => $comment_id)) %>">
-                            <i class="fa fa-fw fa-trash"></i>
-                        </button>
-                    % }
-                    % if (current_user && (current_user->id eq $user->id)) {
-                        <button class="btn btn-primary btn-circle btn-sm trigger-edit-button" type="button" name="editComment" onclick="showCommentEditor(this.form);">
-                            <i class="fa fa-fw fa-pencil"></i>
-                        </button>
-                    % }
-                    <h4 class="media-heading">
-                        % if($comment) {
-                            <%= $user->name %> wrote
-                            <a href="#comment-<%= $comment_id %>" class="comment-anchor"><abbr class="timeago" title="<%= $comment->t_created->datetime() %>Z"><%= format_time($comment->t_created) %></abbr></a>
-                            % if ($comment->t_updated->subtract(seconds => OpenQA::Constants::DB_TIMESTAMP_ACCURACY) >= $comment->t_created) {
-                                (last edited <abbr class="timeago" title="<%= $comment->t_updated->datetime() %>Z"><%= format_time($comment->t_updated) %></abbr>)
-                            % }
+                      <button 
+                        class="btn btn-danger btn-circle btn-sm remove-edit-button" 
+                        hx-delete="<%= url_for($delete_action => ($context->{type}.'_id' => $context->{id}, comment_id => $comment_id)) %>" 
+                        hx-confirm="Do you really want to delete the comment written by <%= $user->name %>?"
+                        hx-target="closest .comment-row, .pinned-comment-row" 
+                        hx-swap="outerHTML"
```

Comment:
I don't think we want to replace the target element with the response. We want to delete the target element - but of course only if the response was a 200 return code. We also need to decrement the number of comments shown in the tab header (if the comment is part of the test details page).

I guess you've chosen a good example here. That *is* the case kind of simple case we might be able to use htmx for. However, if it turns out that even this simple case it too complicated we know that we probably don't want to use htmx.